app/serializers/application.js

```diff
@@ -5,4 +5,7 @@ export default DS.JSONAPISerializer.extend({
   keyForAttribute(attr) {
     return Ember.String.underscore(attr);
   },
+  keyForRelationship: function(key) {
+    return Ember.String.underscore(key);
+  },
```

Comment:
Yeah, this is something we already had with attributes, as you can see. The default for rails is underscore, while the default for ember-data is a dash, I believe. It may have changed recently on one side, though, so I'm not sure anymore. Maybe we should look into it?